routing/payment_session.go

```diff
@@ -246,6 +259,9 @@ func (p *paymentSession) RequestRoute(maxAmt, feeLimit lnwire.MilliSatoshi,
 
 			// Put a lower bound on the minimum shard size.
 			if maxAmt < p.minShardAmt {
+				p.log.Debugf("not splitting because minimum " +
+					"shard amount has been reached")
```

Comment:
nit: log the `minShardAmt` or change to `log.Debug`?

Reply:
variable added